client/src/components/ui/form-field-container.tsx

```
@@ -0,0 +1,84 @@
+interface FormFieldContainerProps {
+  label?: string;
+  hint?: string;
+  error?: string;
+  required?: boolean;
+  validationState?: 'idle' | 'error' | 'success' | 'loading';
+  children: React.ReactNode;
+  className?: string;
+  fieldId?: string;
+}
+
+export const FormFieldContainer = React.forwardRef<HTMLDivElement, FormFieldContainerProps>(
```

ESLint

[error] Replace `HTMLDivElement,·FormFieldContainerProps` with `⏎··HTMLDivElement,⏎··FormFieldContainerProps⏎`

(prettier/prettier)